bedrock/base/templates/macros.html

```diff
@@ -292,13 +292,15 @@ <h3>{{ _('Your download link was sent.') }}</h3>
 
     DO NOT include utm_source in this dictionary! (It's already a required param.)
 #}
-{% macro fxa_email_form(entrypoint, utm_source, style, class_name='fxa-email-form', utm_params={}, form_title='', intro_text='', button_text='', button_class='button button-blue') -%}
+{% macro fxa_email_form(entrypoint, utm_source, style, class_name='fxa-email-form', campaign='', utm_params={}, form_title='', intro_text='', button_text='', button_class='button button-blue') -%}
   {% set service  = 'sync' %}
   <form action="{{ settings.FXA_ENDPOINT }}" data-mozillaonline-action="{{ settings.FXA_ENDPOINT_MOZILLAONLINE }}" id="fxa-email-form" class="{{ class_name }}">
     <input type="hidden" name="action" value="email" />
     <input type="hidden" name="context" value="fx_desktop_v3" />
     <input type="hidden" name="service" value="{{ service }}" />
     <input type="hidden" name="entrypoint" value="{{ entrypoint }}" id="fxa-email-form-entrypoint" />
+    <input type="hidden" name="utm_campaign" value="{{ campaign }}-fxa-embedded-form" id="fxa-email-form-utm-campaign" />
```

Comment:
Justin seems to suggest [in the issue](https://github.com/mozilla/bedrock/pull/7397#issuecomment-509692686) that this value should be `fxa-embedded-form` or `trailhead-fxa-embedded-form`, but we seem to be passing things like `accounts-fxa-embedded-form` or `firefox-new-fxa-embedded-form` which is specific to the page itself. Is that what we want? (I'm not clear here, it may be the case but thought I should ask).

Also, if we omit the `campaign` parameter here the value ends up being `-fxa-embedded-form` (extra `-` at the start, which isn't quite what we need).

Reply:
I'd also suggest ordering these next to `utm_source` which is already hard-coded down on line 310.

Reply:
I think I was confused because of what was already there. I think the standard should be campaign names if applicable: ex: `fxa-embedded-form` or `trailhead-fxa-embedded-form`, and in the future `skyline-embedded-form`.
I will remove all page-specific values. 👍

Reply:
I believe there are only a finite set of allowed utm params. Couldn’t we add term as a parameter? It only seems to be used in one place, and the for loop can lead to a confusing state.